ArduCopter/mode_throw.cpp

```diff
@@ -52,6 +52,7 @@ void ModeThrow::run()
 
     } else if (stage == Throw_Detecting && throw_detected()){
         gcs().send_text(MAV_SEVERITY_INFO,"throw detected - spooling motors");
+        copter.set_land_complete(false);
```

Comment:
My guess is that this is an example where the autotune turned up a situation where the mode was not fulfilling it's new responsibility to set land_complete = false.  Previously it was likely being caught by the overall check in update_land_detector().

I really think we are opening ourselves up to small programming errors in a single mode (which may not be well tested) leading to rare but reputation damaging fall-out-of-the-sky situations.

Reply:
No, I new this was here and didn't require auto test. There are only three ways an aircraft leaves the ground. Manual, takeoff (manual and auto) and throw.